src/lib/core/util/debounce.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Returns a function that won't be invoked, as long as it keeps being called. It will
+ * be invoked after it hasn't been called for `delay` milliseconds.
+ *
+ * @param func Function to be debounced.
+ * @param delay Amount of milliseconds to wait before calling the function.
+ * @param context Context in which to call the function.
+ */
+export function debounce(func: Function, delay: number, context?: any): Function {
```

Comment:
It's been a while since I submitted this PR, but IIRC we can't really use it here because the callback is exposed through an EventEmitter which is consumed through the template; `<div (cdkObserveContent)="someCallback()">`. Technically we could debounce it with Rxjs inside the consumer, but that would make it really inconvenient (we'd need another observable that just proxies this one).

Another alternative for debouncing it through Rxjs could be to have an `exportAs` for the ObserveContent directive and using a ViewChild to access the EventEmitter directly, but that might look weird as well.